content/articles/secondary-dns.markdown

```diff
@@ -7,6 +7,10 @@ categories:
 
 # Secondary DNS
 
+<warning>
+  You cannot set up Secondary DNS if you have [DNSSEC](/articles/dnssec) enabled, as they will not work in conjunction. Please ensure that you are not currently using DNSSEC, or have this feature disabled before using Secondary DNS.
```

Comment:
```suggestion
  You cannot set up Secondary DNS if you have [DNSSEC](/articles/dnssec) enabled, as they will not work in conjunction. Please ensure that you are not currently using DNSSEC, or disable DNSSEC before using Secondary DNS.
```

Reply:
Thank you! And agreed, they should be a separate article that can be linked to from the warning.